prefect_dbt/cloud/clients.py

```diff
@@ -0,0 +1,72 @@
+"""Module containing clients for interacting with the dbt Cloud API"""
+from typing import Optional
+
+import prefect
+from httpx import AsyncClient, Response
+
+from prefect_dbt.cloud.models import TriggerJobRunOptions
+
+
+class DbtCloudAdministrativeClient:
+    """
+    Client for interacting with the dbt cloud Administrative API.
+
+    Args:
+        api_key: API key to authenticate with the dbt Cloud administrative API.
+        account_id: ID of dbt Cloud account with which to interact.
+        domain: Domain at which the dbt Cloud API is hosted.
+    """
+
+    def __init__(self, api_key: str, account_id: int, domain: str):
+        self._closed = False
+        self._started = False
+
+        self._admin_client = AsyncClient(
+            headers={
+                "Authorization": f"Bearer {api_key}",
+                "user-agent": f"prefect-{prefect.__version__}",
```

Comment:
When you're ready to test dry runs with this user-agent, I should see something in the logs come through. 

Looks like the 1.0 prefect user-agent is only coming through for now.

<img width="1084" alt="image" src="https://user-images.githubusercontent.com/19176976/171294333-d62b9458-0118-4430-a05e-aedae5013379.png">

Reply:
I ran a test against the dbt Cloud API so there should be at lease one request with `prefect-2.0b5` as the user-agent header.